users/handlers.py

```diff
@@ -39,20 +39,11 @@ def add_user():
 @app.route('/users')
 @cross_origin(supports_credentials=True)
 def users():
-	lst = []
-	count = 0
+
 	try:
 		user = UserModel()
 		rows = user.get_users()
-		for x in rows:
```

Comment:
handler should not worry about how data is stored in DB. Model should always return dictionary or list of dict. Read more about it here: https://medium.com/@andrew.dewhirst8/model-view-controller-how-to-use-the-mvc-architecture-to-achieve-separation-of-concerns-1042c093f51d#View